Elevel/Application/Features/AuditionFeatures/DeleteAuditionCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Elevel.Application.Interfaces;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Elevel.Application.Features.AuditionFeatures
+{
+   public class DeleteAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            public Handler(IApplicationDbContext context)
+            {
+                _context = context;
+            }
+            public async Task<Response> Handle(Request request, CancellationToken cancelationtoken)
+            {
+                var audiotion = await _context.Auditions.FirstOrDefaultAsync(a => a.Id == request.Id, cancelationtoken);
+                if (audiotion is null)
+                {
+                    return null;
+                }
+                _context.Auditions.Remove(audiotion);
```

Comment:
Actually we don't delete the data from the Audition, we just need to create the property Deleted (such as like in Question entity) and then change its value to true
So it will be some kind of update